### ISSUE TITLE
MD014 incorrectly triggered from empty code block

### ISSUE THREAD

**Issue by ColemanTom** [DavidAnson/markdownlint#1379]
MD014 is about the use of `$` in a code block. But, if you have an empty code block, it is also triggered.

Example markdown file

````
# A markdown file

Hello

```

```
````

Result from mdl

```
$ mdl b.md
b.md:5: MD014 Dollar signs used before commands without showing output
```

### Expectation

Either fail due to a new rule about empty code blocks, or modify MD014 to mention empty code blocks too.

**Comment by ColemanTom** [comment]
Sorry, I just realised, this is the wrong `markdownlint` repo.
